apps/worker/src/bootstrap.ts

```diff
@@ -64,6 +64,9 @@ export async function bootstrap(): Promise<INestApplication> {
   app.use(bodyParser.json());
   app.use(bodyParser.urlencoded({ extended: true }));
 
+  // Starts listening for shutdown hooks
+  app.enableShutdownHooks();
```

Comment:
you told me some time ago to remove this 😛 hahah

Reply:
I don't remember it 🙈 . This is needed for the graceful shutdown hooks to operate after the SIGTERM so I would need the context on why I could have said that. Maybe I mentioned what NestJS says about not optimal performance when enabled it?